### PR TITLE
first commit on branch thomas

### PR DESCRIPTION
**Pull request by robert21in**

### CODE REVIEW

File: branchExercise.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 line 1 branch gartee
-line 2 branch gartee
\ No newline at end of file
+line 2 branch gartee
+line 3 branch thomas
```
Comment (gartee-john-PFG) [inline branchExercise.txt:3]: this looks very good!

**Review by gartee-john-PFG — APPROVED** [review]
it all looks good to me!

**Review by NiveditaPFG — APPROVED** [review]
It looks good to me as well.

**Review by SusmanNickiPFG — APPROVED** [review]
Great work!

**Pull request merged**